astropy/io/fits/tests/test_hdulist.py

```diff
@@ -1040,12 +1040,14 @@ def test_pop_extname(self):
             assert len(hdul) == 5
             assert hdu_popped is hdu1
 
+    # Skip due to ttps://github.com/astropy/astropy/issues/8916
+    @pytest.mark.skipif('sys.platform.startswith("win32")')
     def test_write_hdulist_to_stream(self):
         """
         Unit test for https://github.com/astropy/astropy/issues/7435
-        Ensure that an HDUList can be written to a stream in Python 2
+        Ensure that an HDUList can be written to a stream in Python 3
```

Comment:
I would rather remove the Python altogether because either it's 2 Abs we could remove it or it's 3 and we don't need to mention it. As long as it links to the issue anyone interested can find the context.

Reply:
Good point. It is removed. And while I am at it, I also fixed the grammar. 😸